docs/build.md

```diff
@@ -102,22 +102,5 @@ questions about local OLS installation.
    * To install to develop: `python3 setup.py develop`
    * To build a source distribution: `python3 setup.py sdist`
 
-## Regenerating test data
-All the test does (for the moment) is checking that parsing 10 records from the XML will (1) not crash and (2)
-provide 10 records after parsing. So to regenerate test data, we just have to extract any 10 records (can just be the
-first 10 records) from the ClinVar XML file:
-
-```bash
-CLINVAR_RELEASE="2019-01"  # set the correct one
-zcat ClinVarFullRelease_${CLINVAR_RELEASE}.xml.gz \
-  | awk 'BEGIN {RS="</ClinVarSet>\n\n"; ORS=RS} {print} NR==10 {exit}' \
-  > ClinvarExample.xml
-echo "</ReleaseSet>" >> ClinvarExample.xml
-gzip -c <ClinvarExample.xml >ClinvarExample.xml.gz
-```
-
-Eyeball input & output files to ensure that the ClinVar format has not changed sufficiently enough to render this
-snippet invalid. Then put the generated files into `clinvar-xml-parser/src/test/resources/` directory.
-
```

Comment:
I moved this section into the batch submission instructions, because this is essentially not a "build" instruction, but something you have to do during processing of each batch where ClinVar version changed.

Reply:
Also, the completion of this step requires setting up `CODE_ROOT`, `BATCH_ROOT` and `CLINVAR_VERSION` variables, so all the more reason to keep it with the batch processing instructions.